sdks/npm/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "client-experimentation",
+  "version": "0.0.1",
+  "description": "",
+  "main": "index.js",
```

`npm run build` puts the `index.js` in the `lib` folder for me, so I needed to add this to get it to work:

```suggestion
  "main": "lib/index.js",
```